ui/src/main/java/edu/wpi/grip/ui/DeployController.java

```diff
@@ -170,6 +170,7 @@ protected void onStop() {
    * thread, and it periodically updates the GUI to inform the user of the current status of the
    * deployment.
    */
+  @SuppressWarnings("PMD.AvoidUsingOctalValues")
```

Comment:
Can't you just put this on the one line?
https://github.com/WPIRoboticsProjects/GRIP/pull/620/files#diff-8b72a3e1e29404d24ce3368b64855520R238